README.md

```diff
@@ -28,29 +39,30 @@ Swedbank Pay iOS SDK enables simple embedding of [Swedbank Pay Checkout](https:/
 5. [Test data](#test-data)
 6. [License](#license)
 
-<a name="example-app"></a>
 ## Example app
 
-There is an [example app](https://github.com/SwedbankPay/swedbank-pay-sdk-ios-example-app) written in Swift.
+There is an [example app][example-app] written in Swift.
 
-<a name="requirements"></a>
 ## Requirements
 
 - The SDK requires iOS 9.0+.
-- [Cocoapods](https://guides.cocoapods.org/using/getting-started.html)
-- To use the SDK you must have a "merchant backend" server running. Please refer to the [merchant backend example documentation](https://github.com/SwedbankPay/swedbank-pay-sdk-mobile-example-merchant) on how to set one up.
+- [Cocoapods][cocoapods-doc]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-ios/pullRequest?prid=5645343)